include/motor-control/core/stepper_motor/motor_interrupt_handler.hpp

```diff
@@ -651,18 +728,30 @@ class MotorInterruptHandler {
         hardware.set_step_tracker(
             static_cast<uint32_t>(position_tracker >> 31));
     }
-#ifdef USE_PRESSURE_MOVE
+#ifdef USE_SENSOR_MOVE
     void send_to_pressure_sensor_queue_rear(
         can::messages::BindSensorOutputRequest& m) {
-        std::ignore = sensor_tasks::get_queues()
-                          .pressure_sensor_queue_rear->try_write_isr(m);
+        SensorClientHelper<SensorClient>::send_to_pressure_sensor_queue_rear(
```

Comment:
Instead of doing all of this templated stuff we should just use the same get queues() method, it makes the code much simpler

Reply:
Necessary because not every subsystem that requires MotorInterruptHandler has a SensorClient

Reply:
We don't need to since they're all behind compiler defines. I made a quick commit that rips it out